crates/aggkit-prover-types/src/lib.rs

```diff
@@ -3,12 +3,14 @@ use bincode::{
     DefaultOptions, Options,
 };
 
-pub const FILE_DESCRIPTOR_SET: &[u8] = include_bytes!("generated/aggkit.prover.bin");
+pub mod compat;
 
-#[path = "generated/aggkit.prover.v1.rs"]
 #[rustfmt::skip]
 #[allow(warnings)]
-pub mod v1;
+mod generated;
```

Comment:
`generated` is a bit too generic :). I would argue that `protobuf` or `protoc` or similar would be a better choice.

Reply:
That's the generic name that we're using in every crate that has generated protobuf file. And this folder isn't exposed publically.